src/cmd-buildextend-installer

```diff
@@ -148,12 +148,28 @@ def generate_iso():
                     tarinfo.mode = 0o0644
                 return tarinfo
 
-            # Install binaries from the grub2-efi and shim rpms (installed
-            # in our COSA container). Manually construct the tarball to ensure
-            # proper permissions and ownership
+            tmpimageefidir = os.path.join(tmpdir, "efi")
```

Comment:
Hm but now don't you need to `mkdir` it?

Reply:
Right, done. Squashed commits as its ready for review